docker/set_context.sh

```diff
@@ -0,0 +1,2 @@
+export DOCKER_CONTEXT=$1
```

Comment:
What is this script for?

Reply:
It replaces the `docker context use` command from here: https://docs.microsoft.com/en-us/azure/container-instances/tutorial-docker-compose#deploy-application-to-azure-container-instances. I prefer an environment variable because it only affects the current shell. And it's nice to have the prompt reflect the context switch.